rust_snuba/src/processors/querylog.rs

```
@@ -17,6 +19,11 @@ pub fn process_message(
     _config: &ProcessorConfig,
 ) -> anyhow::Result<InsertBatch> {
     let payload_bytes = payload.payload().context("Expected payload")?;
+    let result: Result<FromQuerylogMessage, _> = utils::from_slice(payload_bytes);
+    match result {
+        Ok(_) => {}
+        Err(err) => println!("ERROR PATH: {}", err.path().to_string()),
```

instead of printing, please return an error here